documentation/FlowManager.md

```diff
@@ -10,7 +10,7 @@ Each "flow" is managed in a different file, with the common events, functions, e
 * **User Login** (`UserLogin.js`) - the flow for a user keying in to the system. The flow is described in details in the [loginAndLogoutFlow](LoginAndLogoutFlow.md) document
 * **User Logout** (`UserLogout.js`) - the flow for a user keying out of the system
 * **Retrieving Settings** (`Settings.js`) - used to retrieve the settings when the system is running in cloud-based mode. See [CloudBasedFlow](CloudBasedFlow.md) for more details
-* **Get User Token** (`GetUserToken.js`) - retrieval of the token of the currently logged in user.
+* **Get GPII Key** (`GetGpiiKey.js`) - retrieval of the GPII key of the currently logged in user.
```

Comment:
This isn't related to the changes in this work, but the points above this one are all incorrect. The files `UserLogin.js`, `UserLogout.js`, and `Settings.js` no longer exist. And the links are broken.